src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayFormat.cs

```diff
@@ -197,6 +197,11 @@ public class SymbolDisplayFormat
                     SymbolDisplayCompilerInternalOptions.FlagMissingMetadataTypes |
                     SymbolDisplayCompilerInternalOptions.IncludeCustomModifiers);
 
+        /// <summary>
+        /// A verbose format for displaying symbols (useful for testing).
+        /// </summary>
```

Comment:
Would it be possible to modify `TestFormat` above instead?

Reply:
> Would it be possible to modify TestFormat above instead?

This is likely to affect a lot of existing tests, because TestFormat doesn't include constraints. Also, it is not clear whether including constraints by default is desired.

---
In reply to: [211791608](https://github.com/dotnet/roslyn/pull/29407#discussion_r211791608) [](ancestors = 211791608)